run.sh will not fail if codewind maven compile fails

**Description:**
If the user tries to build codewind using `./run.sh` and for some reason the users java is not setup properly the mvn package will fail but `./run.sh` continues. This becomes a problem later on as liberty projects don't work.

```
+++   BUILDING DOCKER IMAGES   +++

Building image codewind-pfe-amd64
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building IDC 1.0.0
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ IDC ---
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ IDC ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] skip non existing resourceDirectory /root/codewind/src/pfe/file-watcher/server/test/performance-test-codewind/src/pfe/iterative-dev/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.5.1:compile (default-compile) @ IDC ---
[INFO] Changes detected - recompiling the module!
[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
[INFO] Compiling 24 source files to /root/codewind/src/pfe/file-watcher/server/test/performance-test-codewind/src/pfe/iterative-dev/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] No compiler is provided in this environment. Perhaps you are running on a JRE rather than a JDK?
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```

If the failure is seen above, the `./run.sh` script should terminate as well.

/area iterative-dev
/priority hot

--- a/src/pfe/build
+++ b/src/pfe/build
@@ -1,6 +1,7 @@
-mvn -f iterative-dev/pom.xml clean package -DskipTests | fgrep -v 'Downloaded' | fgrep -v 'Downloading';
-if [ $? -ne 0 ]; then
-    echo "iterative-dev build failed"
+mvn -f iterative-dev/pom.xml clean package -DskipTests
+EC=$?
+if [ $EC -ne 0 ]; then
+    echo "iterative-dev build failed with exit code " $EC
     exit 1;
 fi
 

